kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/filtered-resources/kjars-sources/cloudbalance/src/main/java/org/kie/server/testing/CloudComputer.java

```diff
@@ -74,4 +76,30 @@ public int getMultiplicand() {
         return cpuPower * memory * networkBandwidth;
     }
 
+    public int hashCode() {
+        return new HashCodeBuilder()
+                .append(id)
```

Comment:
I'd argue that if the id is equals, then the computers are equal... (presuming the id isn't null of course).